### PR TITLE
Use institution physical address for free text search

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
As a developer, I need the free text search to reference the physical address for the institution, so that the institution is returned when a user searches for institutions where they would be attending classes.
## Original issue(s)
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/23637

## Testing done
local testing - Using search term `montgomery` looking for Amridge University in list of results

## Acceptance criteria
- [x] When searching for any of the following, the institution physical address fields (street 1, street 2, street 3, city, state, zip, country) are used:
a. city
b. state
c. city, state (abbreviation)
d. zip code
e. country

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/serializers/institution_serializer.rb [hunk 1]
```diff
@@ -27,14 +27,17 @@ class InstitutionSerializer < ActiveModel::Serializer
   attribute :address_1
   attribute :address_2
   attribute :address_3
+  attribute :updated_at
```
Comment (shawkey-daniel) [inline app/serializers/institution_serializer.rb:30]: This is duplicate of line 26

File: app/models/institution.rb [hunk 2]
```diff
@@ -292,14 +297,14 @@ def self.postgres_regex_escape(search_term)
       clause <<  'institution_search LIKE UPPER(:institution_search_term)'
     end
 
-    clause << 'UPPER(city) = :upper_search_term'
+    clause << "UPPER(#{city_physical}) = :upper_search_term"
     clause << 'UPPER(ialias) LIKE :upper_contains_term'
-    clause << 'zip = :search_term'
-    clause << 'country LIKE :upper_contains_term'
+    clause << "#{zipcode_physical} = :search_term"
+    clause << "#{country_physical} LIKE :upper_contains_term"
```
Comment (shawkey-daniel) [inline app/models/institution.rb:303]: Can we keep the single quotes?

File: app/models/institution.rb [hunk 3]
```diff
@@ -292,14 +297,14 @@ def self.postgres_regex_escape(search_term)
...
 
     if include_address
       3.times do |i|
-        clause << "lower(address_#{i + 1}) LIKE :lower_contains_term"
+        clause << "lower(#{address_physical}_#{i + 1}) LIKE :lower_contains_term"
```
Comment (shawkey-daniel) [inline app/models/institution.rb:307]: Can we have single quotes here to be consistent?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: app/serializers/institution_serializer.rb (hunk 1 above)
Comment (ksantiagoBAH) [reply app/serializers/institution_serializer.rb]: Removed

File: app/models/institution.rb (hunk 2 above)
Comment (ksantiagoBAH) [reply app/models/institution.rb:303]: No, the string interpolation wont work with single quotes.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Review by rileyanderson — APPROVED** [review]

**Pull request merged**